src/main/resources/licenses/bsd-3-clause.txt

```diff
@@ -0,0 +1,30 @@
+Copyright (c) <year> <owner>.
```

Comment:
Why the year and owner here are missing?

Reply:
The MIT license in the WebSocket SOAtest extension(`IoT_Microservices_Pack-1.0.4\IoT_Microservices_Pack\com.parasoft.soavirt.transport.websocket-1.0.0-dist\licenses\MIT-license.txt`) also leaves the year and owner to be specified, as `Copyright (c) <year> <copyright holders>`.